pkg/builder/build.go

```diff
@@ -100,42 +100,44 @@ func (b *Builder) WithCapacity(capacity string) *Builder {
 	return b
 }
 
-// WithCompression sets compression of CStorVolumeClaim
-func (b *Builder) WithCompression(compression string) *Builder {
+// WithEncryption sets the encryption on ZFSVolume
+func (b *Builder) WithEncryption(encr string) *Builder {
+	b.volume.Object.Spec.Encryption = encr
+	return b
+}
 
-	comp := "off"
-	if compression == "on" {
-		comp = "on"
-	}
-	b.volume.Object.Spec.Compression = comp
+// WithKeyLocation sets the encryption key location on ZFSVolume
```

Comment:
Is the location/path on the node - absolute or relative path?

Reply:
absolute path. also if someone wants to use keylocation, he has to mount that path to the same path in node-agent.

Reply:
It will be good to provide an example usage for this one in the PR description or the commit message.

Reply:
done.